source/geod24/LocalRest.d

```diff
@@ -820,6 +823,24 @@ public final class RemoteAPI (API, alias S = VibeJSONSerializer!()) : API
         {
             C.send(this.childTid, FilterAPI(""));
         }
+
+        /***********************************************************************
+
+            Call the provided delegate with a scoped copy of this RemoteAPI
+            with a custom timeout.
+
```

Comment:
```suggestion
            Call the provided delegate with a custom timeout

            This allow to perform requests on a client with a different timeout,
            usually to allow some requests (e.g. initialization calls) to have longer
            timeout, or no timeout at all, or to put a timeout on an otherwise
            timeout-less client (e.g. when calling the actual test which could fail).
            
            To disable timeout, pass the special value `Duration.zero`
            (or `0.seconds`, `0.msecs`, etc...).
```